### DISCUSSION TITLE
Give your opinion on the documentation here.

### DISCUSSION THREAD

**Discussion by philibertc** [philibertc/micronova_controller#8] [Q&A]
Hello,
Did you find the documentation clear?
What needs to be improved?
Reply or react below!

**Comment by kaos981** [comment]
Hello, many thanks for the project!
I have a Nordica Extraflame Duchessa Idro and i started to investigate how to communicate with the stove and your information are gems.
Do you know if the adresses  of the variables are the same on all the Micronova stove suppliers?
Many thanks again,
cheers.
Marco

**Reply by philibertc** [reply]
Hello, thanks !
Normally yes, the addresses are the same.
EDIT: 
Most of the addresses are the same,
except one (there may be others): the flue gas temperature address seems to be different quite often, but eventually I'll manage to reference them all

**Reply by testeur70** [reply]
Hello I will try to follow the documentation on github. I will report if I suceed and if it works with m'y Piazzetta stove.
